scripts/fiber/tests-failing.txt

```diff
@@ -42,8 +42,8 @@ src/renderers/dom/shared/__tests__/ReactDOMTextComponent-test.js
 * can reconcile text arbitrarily split into multiple nodes
 * can reconcile text arbitrarily split into multiple nodes on some substitutions only
 
-src/renderers/dom/shared/__tests__/ReactEventListener-test.js
-* should batch between handlers from different roots
+src/renderers/dom/shared/wrappers/__tests__/ReactDOMInput-test.js
+* should control a value in reentrant events
```

Comment:
I don't quite understand this. Why is this failing now? Are we waiting for another fix?